pkg/webservice/handlers.go

```diff
@@ -339,3 +342,55 @@ func getClusterConfig(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
 }
+
+var mutex sync.Mutex
```

Comment:
can we use the RWMutex defined here: https://github.com/apache/incubator-yunikorn-core/blob/742e21611d15da8e3c8d5441f8d3b42d8532634c/pkg/webservice/webservice.go#L41. But looks like we need to move it out of the constructor.

Reply:
Done